Tella/Scenes/Settings/Views/Servers/AddServer/AddServerURLView.swift

```diff
@@ -76,7 +76,7 @@ struct AddServerURLView: View {
         .onAppear {
             
 #if DEBUG
-            serverViewModel.projectURL = "https://api.beta.web.tella-app.org/p/dhekra"
```

Comment:
same here let's not publish any kind of private informations on a public repository.